lib/src/mime_type.dart

```diff
@@ -25,6 +25,19 @@ int get defaultMagicNumbersMaxLength => _globalResolver.magicNumbersMaxLength;
 String lookupMimeType(String path, {List<int> headerBytes}) =>
     _globalResolver.lookup(path, headerBytes: headerBytes);
 
+/// Returns the extension for the given MIME type.
+///
+/// If there are multiple extensions for [mime], return the first occurrence in
+/// the map. If there are no extensions for [mime], return [mime].
+String extensionFromMime(String mime) {
+  for (final key in defaultExtensionMap.keys) {
```

Comment:
may considering doing `in defaultExtensionMaps.entries` – it's a tiny allocation, but then you avoid hitting the content of the map for every key 🤷

Reply:
done